frontend/components/forms/fields/Checkbox/Checkbox.tsx

```diff
@@ -66,28 +68,33 @@ const Checkbox = (props: ICheckboxProps) => {
 
   return (
     <FormField {...formFieldProps}>
-      <label htmlFor={name} className={checkBoxClass}>
-        <input
-          checked={value}
-          className={`${baseClass}__input`}
-          disabled={disabled}
-          id={name}
-          name={name}
-          onChange={handleChange}
-          onBlur={onBlur}
-          type="checkbox"
-        />
-        <span className={checkBoxTickClass} />
-        {tooltipContent ? (
-          <span className={`${baseClass}__label-tooltip tooltip`}>
-            <TooltipWrapper tipContent={tooltipContent}>
-              {children as string}
-            </TooltipWrapper>
-          </span>
-        ) : (
-          <span className={`${baseClass}__label`}>{children} </span>
+      <>
+        <label htmlFor={name} className={checkBoxClass}>
+          <input
+            checked={value}
+            className={`${baseClass}__input`}
+            disabled={disabled}
+            id={name}
+            name={name}
+            onChange={handleChange}
+            onBlur={onBlur}
+            type="checkbox"
+          />
+          <span className={checkBoxTickClass} />
+          {tooltipContent ? (
+            <span className={`${baseClass}__label-tooltip tooltip`}>
+              <TooltipWrapper tipContent={tooltipContent}>
+                {children as string}
```

Comment:
I believe this `as string` type can be removed now that typing has been improved on the `TooltipWrapper`

Reply:
nice!